pymathics/graph/algorithms.py

```diff
@@ -176,23 +176,39 @@ def eval(self, graph, expression, evaluation: Evaluation, options: dict):
 
 class PlanarGraphQ(_NetworkXBuiltin):
     """
+    See <url>https://en.wikipedia.org/wiki/Planar_graph</url>
+
     <dl>
       <dd>PlanarGraphQ[g]
       <dd>Returns True if g is a planar graph and False otherwise.
     </dl>
 
     >> PlanarGraphQ[CycleGraph[4]]
     = True
+
     >> PlanarGraphQ[CompleteGraph[5]]
     = False
+
+    >> PlanarGraphQ[CompleteGraph[4]]
+     = True
+
+    >> PlanarGraphQ[CompleteGraph[5]]
+     = False
+
+    #> PlanarGraphQ[Graph[{}]]
```

Comment:
In new code please let's not use `#>`  these should be pytests. I think there is a `test` directory with a pytest in it as an example.

Reply:
Ok - I see this isn't _new_ code but _moved_ code. But again, please let's get rid of `#>` whenever it occurs. 

(The fact that this has been deprecated since the _last_ release and hasn't been fixed is sad).